### PR TITLE
Revert pr 44

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

File: lib/vsc/install/commontest.py [hunk 1]
```diff
@@ -122,7 +122,7 @@ def _import(self, pkg):
         except ImportError as e:
             log.warn('Import of %s failed with %s' % (pkg, e))
 
-        self.assertTrue(pkg in sys.modules)
+        self.assertTrue(pkg in sys.modules, msg='import %s was success' % pkg)
```
Comment (boegel) [inline lib/vsc/install/commontest.py]: how about doing an `assertTrue(False, ...)` in the `except` above, where you can include the error in the message?

**Comment by stdweird** [comment]
@boegel updated

File: lib/vsc/install/commontest.py (hunk 1 above)
Comment (JensTimmerman) [inline lib/vsc/install/commontest.py]: why not just let the error get raised ?

Comment (stdweird) [inline lib/vsc/install/commontest.py]: ?

**Pull request merged**